requirements.txt

```diff
@@ -54,7 +54,7 @@ scipy==1.7.3
 seaborn==0.11.2
 statsmodels==0.13.1
 strax==1.1.5
-straxen==1.2.4
+straxen==1.2.5
```

Comment:
This change is only to accomodate the new numba, no structural changes that require a new container tag